jbrowse/src/client/JBrowse/utils.ts

```diff
@@ -366,7 +366,8 @@ export async function fetchLuceneQuery(filters, sessionId, trackGUID, offset, pa
             successCallback(jsonRes)
         },
         failure: function(res) {
-            failureCallback("There was an error: " + res.status + "\n Status Body: " + res.responseText + "\n Session ID:" + sessionId)
+            console.log("There was an error: " + res.status + "\n Status Body: " + res.responseText + "\n Session ID:" + sessionId)
```

Comment:
can you make that console.error? there is sometimes page code to catch and report client errors to the server